app/uk/gov/hmrc/exports/mongock/changesets/CacheChangeLog.scala

```diff
@@ -137,6 +137,16 @@ class CacheChangeLog {
     logger.info("Applying 'CEDS-2250 Add one structure level to /transport/borderModeOfTransportCode' db migration... Done.")
   }
 
+  @ChangeSet(order = "007", id = "CEDS-2366 Add Office Of Exit answer", author = "Paulo Monteiro")
```

Comment:
This db migration is not now required, or is it?

Reply:
:facepalm: 
Yes let me remove this one.